CODEOWNERS

```diff
@@ -17,7 +17,6 @@ homeassistant/components/abode/* @shred86
 homeassistant/components/adguard/* @frenck
 homeassistant/components/airly/* @bieniu
 homeassistant/components/airvisual/* @bachya
-homeassistant/components/alarm_control_panel/* @colinodell
```

Comment:
We need to update the manifest file too.

Reply:
The manifest file has been manually updated.  `script/gen_requirements_all.py` did not result in any changes, so I think we should be good here :)